src/contracts/PreAMMBatcher.sol

```diff
@@ -29,6 +29,8 @@ contract PreAMMBatcher {
         address sellToken;
         address buyToken;
         address owner;
+        uint32 validFrom;
+        uint32 validUntil;
```

Comment:
AFAIK, timestamps in Ethereum are `uint64`s, not `uint32`s. Practically speaking, this is not an issue though. But since there is a bunch of dead space anyway because of how Solidity ABI encodes this, might as well make the types large enough.

Reply:
Eventually we plan to encode order parameters tightly, so `uint32` is going to save us sixteen whopping bytes of data per order! :stuck_out_tongue:

Reply:
Yes, but then we don't match the actual EVM datatype... Its not a big deal in this case though, I'll be dead before we run out of u32 :stuck_out_tongue:

Reply:
Just to be safe, I tested how comparisons between different numeric types work. (I couldn't find any reference for this.)

```
pragma solidity ^0.6.12;

contract Comparer {
    function compare(uint32 first, uint64 second) public pure returns (bool) {
        //return first < second;
        return second > first;
    }
}
```

In both cases (commented out and uncommented) it works as expected, so there is no bit trunctation going on before the comparison.
```
ffffffff<100000000? true ✔
2<100000001? true ✔
```